### ISSUE TITLE
Soil data indentification

### ISSUE THREAD

**Issue by ktoddbrown** [ktoddbrown/PrecipOnSOC#2]
ISCN3 has several variables to choose from for soil data including organic carbon as a mass-percentage and bulk density. Most data also has lat-lon-depth-time. What other variables are we interested in for this study?

**Comment by vlbailey** [comment]
definitely OC mass% and Db. How is texture identified? By category (e.g., SiL) or with actual %clay and %sand? Ideally we'd get both. Db plus percent clay would tell me a lot.

Can we get any kind of information about water contents at critical thresholds (Field capacity, PWP)?

**Comment by bpbond** [comment]
Agreed, Db + OC + %clay seems like a good place to start if possible.

**Comment by ktoddbrown** [comment]
We can do clay, sand and silt percentage. Keep in mind that everything we add a new variable we reduce the size of the database. That being said, I think we still have a few thousand profiles to work with!

Field capacity is sadly not in ISCN3 though I can check the NRCS data product to see if there is anything there if we want. We might be able to pull electronic conductivity if that is useful.

**Comment by bpbond** [comment]
Let's not pull unnecessary data. If %sand or %silt reduces the dataset size, I'd drop them–let's focus on clay, which we have _a priori_ reason to think might be important. The other two, not so much.

**Comment by ktoddbrown** [comment]
bulk density, organic carbon, and clay percentage gives us anywhere from 16K to 5K soil cores to work with from ISCN depending on how strict we want to be with our bulk density. If we only want to use sieved oven-dried soils then we are stuck with 5K. If we are ok with un-sieved and some amount of moisture then we have more. 

My inclination is to go with the 5K sample but I wanted your thoughts.

**Comment by bpbond** [comment]
5K seems like a healthy-sized data set. 👍
